### ISSUE TITLE
Counter intuitive exponential backoff 

### ISSUE THREAD

**Issue by jasonjckn** [BrunoBonacci/safely#9]
Counter intuitive exponential backoff 

(take 10 (#'safely.core/exponential-seq 100))
;; => (100 100 100 100 100 100 100 100 100 100)

This had my mind scratching for 5 minutes, wondering if I was using your library wrong, not very exponential. 

Why not 

```
delay = base-delay * 2 ^ (retry-number - 1) +/- random-variation
```

**Comment by BrunoBonacci** [comment]
[corrected]

Indeed it is counterintuitive.... 
The formula I came up with follows the idea: "**Do what I mean, not what I said**" 

In this page there is a full explanation of the formula and the expected behaviour
https://github.com/BrunoBonacci/safely#random-exp-backoff

**Sorry, I misinterpreted your formula and sent you an incorrect response on earlier message**

Indeed your solution works and I will plug it in the next release.

**Comment by BrunoBonacci** [comment]
Released with `1.0.0`
